src/prettify.js

```diff
@@ -1740,3 +1741,14 @@ var prettyPrint;
     });
   }
 })();
+
+// npm require support
+if (module && module.exports) module.exports = {
```

Comment:
Probably it shouldn't be an issue in most cases but the second one would be more consistent.